setup.py

```diff
@@ -116,7 +116,7 @@
             # For render
             "opencv-python",
             # For atari games,
-            "atari_py~=0.2.0",
+            "ale-py~=0.7",
```

Comment:
Could we do 

```
    "ale-py~=0.7.1",
    "autorom[accept-rom-license]~=0.4.2",
```

Reply:
I think we should include autorom yes, otherwise it will be confusing.